src/image-viewer.js

```diff
@@ -316,13 +316,16 @@ export class ImageViewer {
 
   /** @private */
   setupGestures_() {
-    this.gestures_ = Gestures.get(this.image_);
-    this.gestures_.onPointerDown(() => {
+    const gesturesWithoutPreventDefault = Gestures.get(this.image_,
```

Comment:
Could you help me understand why `gesturesWithoutPreventDefault` is needed in this case, but not for `this.gestures_`?

Reply:
So the gesture that's registered here is `onPointerDown`. If we `preventDefault` here, then this gesture recognizer will consume all `onPointerDown` events, including the `click` event that we use in `amp-lightbox-viewer` to register the `toggleControls` function. So unless we're actually using this gesture to halt a moving pan, we still want the event to fire its default so that `toggleControls` will get called. =)